src/parseFunctions.js

```diff
@@ -1,4 +1,4 @@
-import { keyTypes } from './constants.js';
+import { keyTypes, CMCDheaders } from './constants.js';
 
 const parseQueryToJson = (queryString) => {
```

Comment:
If I understood your comment correctly, CMCD headers will contain different keys for each header, so I'm currently using it to check and parse the headers, but it will be even more useful in the future.